Steer/DigitizerWorkflow/src/MIDDigitizerSpec.cxx

```diff
@@ -93,6 +96,11 @@ class MIDDPLDigitizerTask : public o2::base::BaseDPLDigitizer
     for (int collID = 0; collID < irecords.size(); ++collID) {
       // for each collision, loop over the constituents event and source IDs
       // (background signal merging is basically taking place here)
+
+      // Skip digits produced before the first orbit
+      if (irecords[collID].orbit < firstTFOrbit) {
```

Comment:
strictly speaking this seems to be skipping collisions happening before the firstTFOrbit.

Is this fully correct for MID? Could it not happen that there are digits created after the first orbit (due to time of flight) ... originating from collisions happening before?

Reply:
And what happens if the orbit is the same but the bunch crossing of the collision earlier than `FirstSampleTFIR`?

Reply:
@sawenzel : the MID time resolution is better than 25 ns, so we do do not have a dispersion like e.g. the MCH. Also, when a new TF starts, the UL waits for the answer to the HB trigger from all of the boards in the electronics before sending the data.
However, we do have a delay of 90 BCs. This is properly accounted for in the Encoder, and corrected in the Decoder. I can include the protection also at this level if needed.

Reply:
Ok understood. Just wanted to make sure. Thanks for this work !